.github/workflows/eden.yml

```diff
@@ -10,11 +10,13 @@ on:
 
 jobs:
   integration:
-    runs-on: ubuntu-20.04
+    name: Integration test (tpm=${{ matrix.tpm }};${{ matrix.fs }})
+    runs-on: ubuntu-22.04
     strategy:
       fail-fast: false
       matrix:
-        hv: ["kvm", "zfs-kvm"]
+        tpm: ["true", "false"]
+        fs: ["zfs", "ext4"]
```

Comment:
We're going from 2 to 4 tests for each PR and they currently take about 180 minutes. Will all 4 run in parallel?

Reply:
Yes, they are running in parallel runners.